systemtest/src/main/java/io/strimzi/systemtest/resources/KeycloakResource.java

```diff
@@ -0,0 +1,110 @@
+/*
+ * Copyright Strimzi authors.
+ * License: Apache License 2.0 (see the file LICENSE or http://apache.org/licenses/LICENSE-2.0.html).
+ */
+package io.strimzi.systemtest.resources;
+
+import io.strimzi.systemtest.Constants;
+import io.strimzi.systemtest.utils.FileUtils;
+import io.strimzi.systemtest.utils.kubeUtils.controllers.DeploymentUtils;
+import io.strimzi.systemtest.utils.kubeUtils.objects.PodUtils;
+import io.strimzi.systemtest.utils.specific.KeycloakUtils;
+import io.strimzi.test.TestUtils;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.io.IOException;
+
+import static io.strimzi.systemtest.resources.ResourceManager.CR_CREATION_TIMEOUT;
+
+public class KeycloakResource {
+
+    private static final Logger LOGGER = LogManager.getLogger(KeycloakResource.class);
+
+    private static final String KEYCLOAK_OPERATOR_DEPLOYMENT_NAME = "keycloak-operator";
+    private static final String KEYCLOAK_CUSTOM_RESOURCE_NAME = "keycloak";
+    private static final String KEYCLOAK_POD_NAME = "keycloak-0";
+
+    public static void keycloakOperator(String namespace) {
+
+        LOGGER.info("Deploying CRDs for Keycloak Operator in {} namespace", ResourceManager.cmdKubeClient().namespace());
+
+        try {
+            ResourceManager.cmdKubeClient().apply(FileUtils.downloadAndUnzip("https://raw.githubusercontent.com/keycloak/keycloak-operator/master/deploy/olm-catalog/keycloak-operator/9.0.2/keycloak-operator.v9.0.2.clusterserviceversion.yaml"));
```

Comment:
We should keep those URLs in some constants, since it's used on multiple places.

Reply:
and why we need CSV?